README.md

```diff
@@ -322,78 +372,9 @@ Comma 3X should reboot into the 15mph calibration screen.
 
 If you're able to calibrate and then use Comma 3X to use the steering wheel (aka "lat support"), you can clean up the cables and put the covers back on.
 
-At this time, Comma 3X can't use the gas and brake pedals (aka "long support") on TSK vehicles.
+Comma power (ODB2+RJ45) is optional, so you can keep it or remove it. Keeping it allows Comma 3X to stay on when you turn off the car, which allows you to upload logs and SSH in more easily. For most people this is unnecessary.
```

Comment:
Maybe we should say that temporary installation (IE. not tucked into the car) might be a good idea during initial setup.

Reply:
Well this is after a successful lat support, so I think they're done with it?

Reply:
Oh I see, nevermind